### PR TITLE
Update from net5.0 to 6.0

### PR DESCRIPTION
**Pull request by timburrows**
Add support for net6.0 projects to maintain compatibility with HotChocolate 13.x.x

### CODE REVIEW

**Comment by MartinRisk** [comment]
@StefH Please create a new release from the pull request of @timburrows 
The current package is not useable for HotChocolade 13

**Comment by StefH** [comment]
I will review the PR next week and merge and create a new release.

File: Directory.Build.props [hunk 1]
```diff
@@ -14,11 +14,11 @@
         <PackageLicenseExpression>MIT</PackageLicenseExpression>
         <PackageReleaseNotes>$([System.IO.File]::ReadAllText("$(MSBuildProjectDirectory)/../../PackageReleaseNotes.txt"))</PackageReleaseNotes>
         <PackageReadmeFile>PackageReadme.md</PackageReadmeFile>
-        <PackageProjectUrl>https://github.com/StefH/HotChocolate.Data.Extensions</PackageProjectUrl>
+        <PackageProjectUrl>https://github.com/StefH/GistLens.HotChocolate.Data.Extensions</PackageProjectUrl>
```
Comment (StefH) [inline Directory.Build.props:17]: Please revert this change to GistLens

File: GistLens.HotChocolate.Data.Extensions - solution.sln [hunk 2]
```diff
@@ -18,7 +18,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{CDA9510C-CE4
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "examples", "examples", "{4AA50138-208E-43B4-8A77-AA4FC0F36000}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "HotChocolate.Data.Filters.Extensions", "src\HotChocolate.Data.Filters.Extensions\HotChocolate.Data.Filters.Extensions.csproj", "{26351237-8299-476A-8051-D3C3F3125F45}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GistLens.HotChocolate.Data.Filters.Extensions", "src\GistLens.HotChocolate.Data.Filters.Extensions\GistLens.HotChocolate.Data.Filters.Extensions.csproj", "{26351237-8299-476A-8051-D3C3F3125F45}"
```
Comment (StefH) [inline GistLens.HotChocolate.Data.Extensions - solution.sln:21]: Revert this change.

**Comment by StefH** [comment]
There are too many changes which are not needed.

I'll close this PR and create a new one in the next days.
